bazel/examples/debian.BUILD

```diff
@@ -0,0 +1,15 @@
+load("@bazel_tools//tools/build_defs/docker:docker.bzl", "docker_build")
+
+# Extract .xz files
+genrule(
```

Comment:
Possibly, I don't know. This is just taken directly from the Bazel documentation. If we/users find some improvement to the Docker workflow in Bazel, I think we'd have more impact by moving that into Bazel's Docker support directly.